engine/crates/composition/src/compose/object.rs

```diff
@@ -97,6 +99,32 @@ pub(super) fn merge_field_arguments<'a>(
     ids.unwrap_or(federated::NO_INPUT_VALUE_DEFINITION)
 }
 
+fn compose_field_argument_defaults(
+    arguments: &[(StringId, subgraphs::FieldArgumentWalker<'_>)],
+    ctx: &mut Context<'_>,
+) {
+    let mut default: Option<(&subgraphs::Value, subgraphs::FieldArgumentWalker<'_>)> = None;
+
+    for (_, argument) in arguments {
+        let Some(value) = argument.default() else { continue };
```

Comment:
If subgraph A doesn't have a default and subgraph B does, should that be an error?  This looks like it'd allow it (though correct me if I'm misunderstanding)